src/pages/taskHistory/BuildVariantSelector.tsx

```diff
@@ -44,49 +42,43 @@ const BuildVariantSelector: React.FC<BuildVariantSelectorProps> = ({
       taskName,
     },
   });
+  const buildVariants = data?.buildVariantsForTaskName ?? [];
 
   const onChange = (selectedBuildVariants: string[]) => {
     updateQueryParams({
       [HistoryQueryParams.VisibleColumns]: selectedBuildVariants,
     });
   };
 
-  const { buildVariantsForTaskName } = data || {};
-
-  const handleSearch = (options: BuildVariantTuple[], match: string) =>
-    options.filter(
-      (option) =>
-        option.buildVariant.includes(match) ||
-        option.displayName.includes(match)
-    );
   return (
-    <Container>
-      <SearchableDropdown
-        data-cy="build-variant-selector"
-        label="Build Variant"
-        valuePlaceholder="Select Build Variant to View"
-        value={value}
-        onChange={onChange}
-        options={buildVariantsForTaskName}
-        disabled={loading}
-        allowMultiSelect
-        searchFunc={handleSearch}
-        optionRenderer={(option: BuildVariantTuple, onClick, isChecked) => (
-          <SearchableDropdownOption
-            key={`searchable_dropdown_option_${option.buildVariant}`}
-            value={option.buildVariant}
-            displayName={option.displayName}
-            onClick={() => onClick(option.buildVariant)}
-            isChecked={isChecked(option.buildVariant)}
-          />
-        )}
-      />
-    </Container>
+    <StyledCombobox
+      data-cy="build-variant-selector"
+      label="Build Variant"
+      placeholder="Select Build Variant to View"
+      disabled={loading}
+      overflow="scroll-x"
+      multiselect
+      onChange={onChange}
+      value={value}
+      clearable
+    >
+      {buildVariants.map(({ displayName, buildVariant }) => (
+        <ComboboxOption
+          key={`combobox_option_${buildVariant}`}
+          data-cy="combobox-option"
+          value={buildVariant}
+          displayName={displayName}
+        />
+      ))}
+    </StyledCombobox>
   );
 };
 
-const Container = styled.div`
-  width: 300px;
+const StyledCombobox = styled(Combobox)`
+  width: 350px;
+  [role="option"]:first-of-type {
+    margin-left: ${size.xs};
+  }
```

Comment:
If i had to guess the same bug from the placeholder is happening here and you are applying styling to fix the padding. I would normally caution away from doing this. Applying styles to the internal workings of third party components can be error prone because we have no guarantee the component won't change. It would be really easy for this to break or some other unintended behavior to occur.  I would rather leafygreen fix this on their side since it is a bug with their component. I raised it in a leafygreen thread. 
https://mongodb.slack.com/archives/CFFA5BS79/p1648674190210719

Reply:
As an aside if you are feeling up to it i'm sure they would appreciate a pr. 👀  https://github.com/mongodb/leafygreen-ui/blob/b57f0da0a1ecffd0dacc9dff9a3b37f1e3658958/packages/combobox/src/Combobox.styles.ts#L158